### ISSUE TITLE
dnsmasq 1.4 fails to start

### ISSUE THREAD

**Issue by ocalvo** [home-assistant/addons#1850]
## The problem
Add-on fails to start with error:
```
[FATAL tini (6)] exec /init failed: Permission denied
```

## Environment

- Add-on with the issue: dnsmaq
- Add-on release with the issue: 1.4
- Last working add-on release (if known): 1.3
- Operating environment (OS/Supervised): hassos

## Problem-relevant configuration

```yaml
defaults:
  - 192.168.1.1
forwards: []
hosts:
  - host: dc.mydomain.io
    ip: 192.168.1.2
services: []
```

## Traceback/Error logs

```txt
[FATAL tini (6)] exec /init failed: Permission denied
```

**Comment by SciTii** [comment]
Same problem...

**Comment by segoz** [comment]
Same problem here

**Comment by TripitonPT** [comment]
Same problem here...

**Comment by boehmap** [comment]
Ditto

**Comment by SciTii** [comment]
Please reapir ir quickly.. My father will hate me cause now he has to turn off wifi every time he wants to log to hassio from inside

**Comment by mattrude** [comment]
I am seeing the following error while trying to start the dnsmasq addon:

```
404 Client Error for http+docker://localhost/v1.40/containers/create?name=addon_core_dnsmasq: Not Found ("No such image: homeassistant/armv7-addon-dnsmasq:1.3")
```

The addon is unable to start after the latest upgrade.

---

## System Health

version | core-2021.2.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
virtualenv | false
python_version | 3.8.7
os_name | Linux
os_version | 5.4.83-v7l
arch | armv7l
timezone | America/Chicago

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Hass.io</summary>

host_os | Home Assistant OS 5.11
-- | --
update_channel | stable
supervisor_version | supervisor-2021.02.6
docker_version | 19.03.13
disk_total | 57.8 GB
disk_used | 7.7 GB
healthy | true
supported | true
board | rpi4
supervisor_api | ok
version_api | ok
installed_addons | Let's Encrypt (4.11.0), Mosquitto broker (5.1), NGINX Home Assistant SSL proxy (3.0.1), Check Home Assistant configuration (3.6.0), Dnsmasq (1.4), SSH & Web Terminal (8.0.1), Z-Wave JS (0.1.6), Z-Wave JS to MQTT (0.4.1), Node-RED (8.1.1)

</details>

<details><summary>Lovelace</summary>

dashboards | 3
-- | --
resources | 0
views | 4
mode | storage

</details>

**Comment by mattrude** [comment]
### One resolution

Seeing that commit 18c939c546787c2225d77e4d486856dc2235dd0d has been reverted by  2b14bd20fa6800e38936a624b10a3212ba192e9e. I backed up my configuration (copied the contents of the plugins configuration tab), uninstalled the dnsmasq addon, reinstalled it, and restored my config.

The dnsmasq addon is working once again on my system.

It is currently labeled as version 1.3

**Others may have to try the same thing to get their networks running again.**

**Comment by SciTii** [comment]
yeah my addon actually backed to 1.3 somehow and works again

**Comment by arjanhs** [comment]
When reinstalling the addon i got:
21-02-16 05:44:45 ERROR (MainThread) [supervisor.utils.apparmor] To many profiles inside file: {'/usr/sbin/dnsmasq', 'dnsmasq'}

**Comment by BartEngelen** [comment]
After upgrade to 1.4 I can not start Dnsmasq.
It says Failed to start addon: Port 53 is already in use by something else on the host.
Version 1.3 works great.
I don't have anything else running on port 53 on this host.

**Comment by pvizeli** [comment]
Fix with 1.4.3
